autoprotocol/protocol.py

```diff
@@ -6745,10 +6748,109 @@ def location_helper(source, destination, volume, method, density):
                 ),
             ]
 
+        def informatics_helper(informatics, dest, multi_src):
+            """
+            Checks Informatics against the Instruction param values, and split
+            Informatics per destination well as needed.
+
+            Parameters
+            ----------
+            informatics: list(Informatics)
+                list of Informatics provided by the user input
+            dest: WellGroup
+                destination wells that could be affected by the instruction
+            multi_src: boolean
+                True if there are multiple sources in the transfer
+
+            Returns
+            -------
+            list(Informatics)
+                list of Informatics per destination well
+
+            Raises
+            ------
+            TypeError
+                Informatics provided is not valid or not supported
+            ValueError
+                Informatics wells must match wells in Instruction
+            ValueError
+                Multiple instances of Informatics on a Well
+            ValueError
+                Parsed list of Informatics length should match the length of destination
+            """
+            if multi_src:
+                dest_count = 1
+            else:
+                dest_count = len(dest)
+
+            if len(informatics) == 1:
```

Comment:
ok, so this is the case where we try to intelligently parse potential intentions

Reply:
I'll add more documentations :)